assets/queries/cloudFormation/aws/api_gateway_access_logging_disabled/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+    "id": "999ecb35-10df-4d73-8f17-3f4b8c3beec5",
+    "queryName": "API Gateway Access Logging Disabled",
+    "severity": "MEDIUM",
+    "category": "Observability",
+    "descriptionText": "API Gateway should have Access Log Settings defined",
+    "descriptionUrl": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-stage.html",
```

Comment:
The descriptionurl should follow the same guidelines as the one from terrafom, and point to the access log settings section of the page. I propose swapping it for this one:  https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-apigateway-stage-accesslogsetting.html

What do you think?

Reply:
Thank you!  Done here: https://github.com/Checkmarx/kics/pull/6863/commits/d1755a29ab548009abd8f01eee9714712a173904